test/testUtils/PlayUtils.scala

```diff
@@ -0,0 +1,11 @@
+package testUtils
+
+import play.api.{Application, GlobalSettings}
+import play.api.test.FakeApplication
+
+object PlayUtils {
```

Comment:
I'm not sure if it's a useful test tho... https://github.com/guardian/subscriptions-frontend/pull/39/files#r32737467